### PR TITLE
Add Padlet

### LINKED ISSUE #1734: Padlet | padlet.com
**Prerequisites and essential questions** 
- [X] Is it a popular site?
- [X] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** geneva#0058 

**What's the name of the service?** Padlet 

**What should the Presence display?** Padlet, Padlet name and time used to view Padlet 

**If possible, please provide a logo for the service (512x512 minimum)**  
image

### PR DESCRIPTION
**Pull request by TheGamerzs**

### CODE REVIEW

File: websites/P/Padlet/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "url": "padlet.com",
+  "description": {
+    "en": "From your hobby to your career, your class notes to your final exam, your mood board to your runway show, padlets help you organize your life."
+  },
+  "settings": [
+    {
+      "id": "buttons",
+      "title": "Show Buttons",
+      "value": true
+    }
```
Comment (Bas950) [inline websites/P/Padlet/dist/metadata.json:23]: icon?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/P/Padlet/dist/metadata.json (hunk 1 above)
Comment (TheGamerzs) [reply websites/P/Padlet/dist/metadata.json:23]: icon you?

File: websites/P/Padlet/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "padlet",
+  "author": {
+    "name": "The Gamerzs",
+    "id": "730142125181894657"
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/sxpYxcR.png",
+  "thumbnail": "https://i.imgur.com/9nXPXZh.png",
```
Comment (EncryptedDev) [inline websites/P/Padlet/dist/metadata.json:10]: Thumbnail should be a screenshot of the homepage or a promotional banner, neither of which this meets

File: websites/P/Padlet/presence.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    //Dashboard
+
+    switch (pathnames[2]) {
+      case "drive":
+        presenceData.details = "Viewing";
+        presenceData.state = `Padlet Drive`;
+        break;
+      case "settings":
+        presenceData.details = null;
+        presenceData.state = null;
```
Comment (EncryptedDev) [inline websites/P/Padlet/presence.ts:23]: no null, use delete

File: websites/P/Padlet/presence.ts [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  }
+
+  const showButtons = await presence.getSetting("buttons");
+
+  if (!showButtons) delete presenceData.buttons;
+
+  if (presenceData.details === null) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else presence.setActivity(presenceData);
```
Comment (EncryptedDev) [inline websites/P/Padlet/presence.ts:101]:
```suggestion
  if (!presenceData.details) {
    presence.setTrayTitle();
    presence.setActivity();
  } else presence.setActivity(presenceData);
```
or
```suggestion
presence.setActivity(presenceData);
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Comment by TheGamerzs** [comment]
image
image

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**